### DISCUSSION TITLE
Botnet code - please remove

### DISCUSSION THREAD

**Discussion by roxchatdev** [community/community#64597] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

We found data for botnets posted at Github
here: https://github.com/db1000n-coordinators
Please, remove it ASAP.

**Comment by queenofcorgis** [comment]
Hi @roxchatdev 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
